api/provider/aws/test_aws/environment_create_test.go

```diff
@@ -39,10 +40,11 @@ func TestEnvironmentCreate(t *testing.T) {
 
 	req := models.CreateEnvironmentRequest{
 		EnvironmentName:  "env_name",
-		InstanceSize:     "m3.small",
+		InstanceType:     "m3.small",
```

Comment:
I don't believe that m3.small is a valid instance type anymore.

https://aws.amazon.com/ec2/instance-types/